pyexcel_xlsxw/xlsx.py

```diff
@@ -39,11 +39,11 @@ def __init__(self):
     def open(self, file_name, **keywords):
         BookWriter.open(self, file_name, **keywords)
         _date_format = "default_date_format"
-        options = {
-            _date_format: 'dd/mm/yy',
-            'constant_memory': True
-        }
+        options = keywords.get('options', {})
```

Comment:
I would suggest to remove 'options' keyword, but instead using keyword as options itself. Your implementation is alright but just because I have used keywords as direct options in other plugins, it will be good to keep the consistency.